R/autoplot.anosim.R

```diff
@@ -0,0 +1,44 @@
+##' @title ggplot-based plot for objects of class \code{"anosim"}
+##'
+##' @description
+##' Produces a multi-layer ggplot object representing the output of objects produced by \code{\link[vegan]{anosim}}.
+##'
+##' @param object an object of class \code{"anosim"}, the result of a call to \code{\link[vegan]{anosim}}.
+##' @param notch logical; make notched (default) or standard box plot?
+##' @param varwidth logical; make box width proportional to to the square-roots of the number of observations in the groups (default) or standart box plot?
+##' @param xlab character; label for the x-axis.
+##' @param ylab character; label for the y-axis.
+##' @param subtitle logical; should subtitle with R and p values be added.
+##' @param ... additional arguments passed to other methods.
+##' @return A ggplot object.
+##' @author Didzis Elferts
+##'
+##' @export
+##'
+##' @importFrom ggplot2 ggplot autoplot geom_boxplot aes_string labs fortify
+##'
+##' @examples
+##'
+##' data(dune)
+##' data(dune.env)
+##' dune.dist <- vegdist(dune)
+##' attach(dune.env)
```

Comment:
`attach()` is bad and to be avoided &mdash; at the very least it should be `detach()`ed at the end of the example. I would prefer `with(dune.env, anosim(dune.dist, Management))`. I suspect this is directly from `?anosim` in which case I should go and update that example in the vegan sources.

Reply:
Yes, that is from anosim example. I will change it with your code.